### DISCUSSION TITLE
Working on a FTXS20D3VMW interior unit

### DISCUSSION THREAD

**Discussion by PabloMartin97** [revk/xESP32-Faikout#582] [Show and tell]
Just installed faikin on a 2008 model FTXS20D3VMW indoor unit and it is working really fine!

All functions works fine including night mode except for powerful mode, which doesn't worry me. 

I hope this helps someone, I was doubtful it would work on such old units. Now it's time to integrate it into HA. 

If there is a place where the compatible models are listed, please let me know so I can post it. I have been unable to find it.

Thanks a lot Revk from Spain!!

[image](https://github.com/user-attachments/assets/4bdbddd6-93a9-4778-acff-8e946a183f4e)

**Comment by revk** [comment]
There is a wiki page you can edit.

**Comment by revk** [comment]
And thank you for feedback
